From b677a9f2d252c8ed4754917ff120589e225adf7e Mon Sep 17 00:00:00 2001
From: Max Krummenacher <max.krummenacher@toradex.com>
Date: Wed, 3 Jan 2018 13:54:04 +0100
Subject: [PATCH] egl: Add EGL_PLATFORM_X11_KHR alias to EGL_PLATFORM_X11_EXT

New EGL code uses EGL_PLATFORM_X11_KHR (for example glmark2) and the
value is the same, so we can provide an alias for it instead of
patching every code using it.

Upstream-Status: Pending

(Adapted from meta-freescale, commit e1d249bb067aa6c472367cff1e77ba2ddbc8303f)

Signed-off-by: Max Krummenacher <max.krummenacher@toradex.com>
---
 EGL/eglext.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/EGL/eglext.h b/EGL/eglext.h
index 80b539a..eb349c2 100644
--- a/EGL/eglext.h
+++ b/EGL/eglext.h
@@ -445,6 +445,7 @@ EGLAPI EGLSurface EGLAPIENTRY eglCreatePlatformPixmapSurfaceEXT (EGLDisplay dpy,
 #ifndef EGL_EXT_platform_x11
 #define EGL_EXT_platform_x11 1
 #define EGL_PLATFORM_X11_EXT              0x31D5
+#define EGL_PLATFORM_X11_KHR              EGL_PLATFORM_X11_EXT
 #define EGL_PLATFORM_X11_SCREEN_EXT       0x31D6
 #endif /* EGL_EXT_platform_x11 */
 
-- 
2.13.6

